src/client/public/detail.html

```diff
@@ -0,0 +1,42 @@
+<div class="form-horizontal col-sm-6">
+    <form name="manageForm" autocomplete="off">
+        <h2>{{action}}</h2>
+        <div class="form-group" ng-class="{error: manageForm.name.$invalid && !manageForm.name.$pristine}">
+            <label for="name" class="col-sm-2 control-label">Name</label>
+            <div class="col-sm-10">
+                <input class="form-control" type="text" name="name" ng-model="editingList.list.name" required placeholder="List name">
+            </div>
+            <span ng-show="manageForm.name.$error.required && !manageForm.name.$pristine" class="help-inline">
```

Comment:
Nice addition. Validation is always nice to have on the front as well as the back end.